lib/attribute_normalizer/normalizers/postal_code_normalizer.rb

```diff
@@ -2,7 +2,7 @@ module AttributeNormalizer
   module Normalizers
     module PostalCodeNormalizer
       def self.normalize(value, options)
-        GsubNormalizer.normalize value, pattern: /\W+|_/, replacement: ''
+        GsubNormalizer.normalize value.upcase, pattern: /\W+|_/, replacement: ''
```

Comment:
can you add a new test? https://github.com/amaabca/attribute_normalizer-extras/blob/master/spec/models/postal_code_normalizer_spec.rb#L17